src/process.rs

```diff
@@ -77,10 +77,13 @@ impl Native {
         cmd: impl AsRef<Path>)
         -> Native
     {
+        let mut command = Command::new(cmd.as_ref());
+        command.env("LANG", "C");
+        command.env("LC_ALL", "C");
```

Comment:
Can we detect if C.utf8 is present and use that?

Reply:
Well, yes. Reading logs is a big ugly with locale `C`. But what to do if users current locale/terminal is not utf8? Should we check current locale first? (also there is no other `C` except `C` and `C.utf8` so we can't transpose any encoding on `C`)